hamilton/function_modifiers/__init__.py

```diff
@@ -29,6 +29,7 @@
 
 # The config decorator
 config = configuration.config
+ignore = configuration.ignore.ignore()
```

Comment:
I think you can skip one level of `ignore` here. E.G. just instantiate the class... It carries no state so it should be good